src/test/components/__snapshots__/ListOfPublishedProfiles.test.js.snap

```diff
@@ -731,7 +731,7 @@ exports[`ListOfPublishedProfiles still displays the link to all recordings when
     <a
       href="/uploaded-recordings/"
     >
-      Manage this recording
+      { $ uploadedProfileInformationListLength } Manage these recordings
```

Comment:
As also discussed, this test snapshot shouldn't be like this. After removing the space I believe it should work as expected.

But this snapshot makes me wonder if the `Localized` component doesn't work. Because this is the fallback text and that fallback text shouldn't be printed if it find the string in the ftl file. It requires a manual testing. When you change the text in the ftl file, does it show up here as well?

Reply:
I'm also thinking that `Localized` is not working here if it's a fallback. I think there no fetching happening here. And I did a mistake about `{ $ uploadedProfileInformationListLength }`. I actually don't have to include it in the string.